### PR TITLE
Release/view and share

### PR DESCRIPTION
**Pull request by princesseana**
1. Readme and tutorial.md - maybe check if too much repetition
2. bug to fix : for the shared data, only the shared stream should appear and not the title of the other stream (index.html) ?

### CODE REVIEW

File: view-and-share/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| Sign in                                                 | Visualize data                                                  | Share data                                                      |
+| ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| <img src="images/1-login.png" alt="base" style="zoom:33%;" /> | <img src="images/2-visualize.png" alt="consent" style="zoom:33%;" /> | <img src="images/3-sharing.png" alt="form" style="zoom:33%;" /> |
+
+## Project Specifications
+
+- Ask for login
+- Request access for the app "pryv-example-view-and-share" to read the streams "Baby" and "Body"
+- Display the data
+- Create or delete a sharing (URL or email) to a third party
```
Comment (kebetsi) [inline view-and-share/README.md:20]: URL or email??

File: view-and-share/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+</p>
+
+
+With click on the login button, a popup opens in your browser where you can either authenticate or create a new account. 
+
+When signed in, you can consent to give the app access to the streams "Body" and "Baby" where the data from the form (see [tutorial "Collect survey data"](collect-survey-data/tutorial.md)) is stored.
+<p align="center">
+<img src="images/request-permissions.png" alt="request-permissions" width=250 />
+</p>
+Once you have accepted, it displays the saved data of your baby's weight and your own blood pressure.
```
Comment (kebetsi) [inline view-and-share/tutorial.md:18]: maybe add a small view with displayed data, from an account that has 1-2 entries of each so it fits the guide conveniently?

File: view-and-share/script.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+var service = null; // will be initialized after setupAuth;
+var username = null; // will be inialized after AUTHORIZED auth State is received
+window.onload = async (event) => {
+  
+  if (apiEndpoint) { // if apiEndpoint then we are in "View only mode"
+    document.getElementById('welcome-message-mme').style.visibility = 'hidden';
+    document.getElementById('please-login').style.visibility = 'hidden';
+    document.getElementById('data-view').style.visibility = 'visible';
+    document.getElementById('welcome-message-viewer').style.visibility = 'visible';
+    document.getElementById('username').innerText = apiEndpoint.split('@')[1].slice(0,-1);
```
Comment (kebetsi) [inline view-and-share/script.js:63]: there might be an issue, it cuts the last param of the API endpoint when provided without the trailer`/`: https://api.pryv.com/app-web-examples/view-and-share/?pryvApiEndpoint=https://ckbry9n6h009o1od3qiv3qz7u@mariana.pryv.me

File: view-and-share/script.js [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+    if (event.streamIds.includes('baby-body') && event.type === 'mass/kg') { // get 'mass/kg' events from the stream 'Baby-Body'
+      addTableEvent(babyDataTable, event, [event.content + ' Kg']);
+      babyDataTable.style.visibility = 'visible';
+    }
+    if (event.streamIds.includes('heart') && event.type === 'blood-pressure/mmhg-bpm') { // get 'blood-pressure/mmhg-bpm' events from the stream 'Heart'
+      addTableEvent(heartDataTable, event, [event.content.systolic + 'mmHg', event.content.diastolic + 'mmHg']);
+      heartDataTable.style.visibility = 'visible';
+    }
+  }
+  if (! apiEndpoint) // display sharings only when logged-in
```
Comment (kebetsi) [inline view-and-share/script.js:108]: I think we can remove this check

File: view-and-share/README.md (hunk 1 above)
Comment (princesseana) [reply view-and-share/README.md:20]: I was specifying the way to share data (clicking on a link or sending the link by email) but it's not clear, so I will only keep "Create or delete a sharing to a third party"

**Comment by kebetsi** [comment]
Can you change all the links hostnames from `pryv.github.io` to `api.pryv.com`?

**Pull request merged**